src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+of bandwidth, as most of the time in establishing a connection is spent waiting,
+rather than exchanging data.
+
+Informing the browser of your intention is as simple as adding a link tag to
+your page:
+
+    <link rel="preconnect" href="https://example.com">
+
+In this case, we’re letting the browser know that we intend to connect to
+`example.com` and retrieve content from there.
+
+Bear in mind that while `<link rel="preconnect">` is pretty cheap, it can still
+take up valuable CPU time, particularly on secure connections. This is
+especially bad if the connection isn’t used within 10 seconds, as the browser
+will close it, wasting all of that early connection work.
+
+In general, try to use `<link rel="preload">` wherever you can, as it’s a more
+comprehensive performance tweak, but do keep `<link rel="preconnect">` in your
+toolbelt for the edge cases. Let’s look at a couple of them.
+
+Note: There’s actually another `<link>` type related to connections:
+`<link rel="dns-prefetch">`. This handles the DNS lookup only, so it’s a small
+subset of `<link rel="preconnect">`, but it’s got wider browser support, so it
+may serve as a nice fallback.
+You use it the exact same way:
+`<link rel="dns-prefetch" href="https://example.com">`
+
+### Use-case: Knowing *Where From*, but not *What* You're Fetching
+
+Due to versioned dependencies, you sometimes end up in a situation where you
+know you’ll be retrieving a resource from a given CDN, but not the exact path
+for it. In other cases, one of several resources may be retrieved, depending
+on media queries or runtime feature checks on the user’s browser.
+
+In these situations, and if the resource you’ll be fetching is important, you
+may want to save as much time as possible by pre-connecting to the server. The
+browser won’t begin fetching the file before it needs it (that is, once the
+request is made from your page somehow), but at least it can handle the
+connection aspects ahead of time, saving the user from waiting for several
+roundtrips.
+
+### Use-case: Streaming Media
+
+Another example where you may want to save some time in the connection phase,
+but not necessarily start retrieving content right away, is when streaming media
+from a different origin.
+
+Depending on how your page handles the streamed content, you may want to wait
+until your scripts have loaded and are ready to process the stream. Preconnect
+helps you cut the waiting time to a single roundtrip once you’re ready to
+start fetching.
+
+## Prefetch
+
+`<link rel="prefetch">` is somewhat different from `<link rel="preload">` and
+`<link rel="preconnect">`, in that it doesn’t try to make something critical
+happen faster; instead, it tries to make something non-critical happen earlier,
+if there’s a chance.
+
+It does this by informing the browser of a resource that is expected to be
+needed as part of a future navigation or user interaction, i.e., something that
```

Comment:
Suggestion to replace i.e. with for example.

Reply:
Done.